src/scss/settings/_variables.scss

```diff
@@ -97,28 +97,22 @@ $responsive-gutters           : true !default; // Disable this if you don't want
 $use-legacy-grid              : true !default;
 
 
-// Spacing (Vertical / Horizontal)
-// ==========================================================================
-$spacing: 8px; // only for use in map below – not for in components
-
 // spacing map
-// Can access levels by using the spacing function > spacing() or spacing(x5) for example
+// Can access levels by using the spacing function > spacing() or spacing(a) for example
 $spacing: (
-    x0.5 : $spacing / 2,
-    base : $spacing,
-    x1.5 : $spacing * 1.5,
-    x2   : $spacing * 2,
-    x3   : $spacing * 3,
-    x4   : $spacing * 4,
-    x5   : $spacing * 5,
-    x6   : $spacing * 6,
-    x7   : $spacing * 7,
-    x8   : $spacing * 8,
-    x9   : $spacing * 9,
-    x10   : $spacing * 10
+    a    : $spacing-a,
+    b    : $spacing-b,
+    c    : $spacing-c,
+    d    : $spacing-d,
+    e    : $spacing-e,
+    f    : $spacing-f,
+    g    : $spacing-g,
+    h    : $spacing-h,
+    i    : $spacing-i,
+    j    : $spacing-j
 );
 
-$baseline: spacing(x2); // default spacing between blocks
+$baseline: spacing(d); // default spacing between blocks
```

Comment:
I think we should look to remove `$baseline` and replace it with the equivalent `$spacing(d)` value now where it's used in fozzie

Reply:
Fixed.